server/src/main/java/com/broadcom/lsp/domain/cobol/event/model/RunAnalysisEvent.java

```diff
@@ -22,7 +22,15 @@
  * all the registered documents.
  */
 public class RunAnalysisEvent extends DataEvent {
+  public final boolean quiet;
```

Comment:
I don't see where do you check this value

Reply:
Oops, you are right. I must use it in `MyTextDocumentService:observerCallback`.